org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/util/JRTUtil.java

```diff
@@ -70,7 +71,7 @@ public class JRTUtil {
 	public static final int NOTIFY_ALL = NOTIFY_FILES | NOTIFY_PACKAGES | NOTIFY_MODULES;
 
 	// TODO: Java 9 Think about clearing the cache too.
```

Comment:
This comment is obsolete if a SoftReference is used. We could even consider to use a smarter cache with a time-based LRU eviction policy.